responses/de/HassLightSet.yaml

```diff
@@ -2,7 +2,5 @@ language: de
 responses:
   intents:
     HassLightSet:
-      brightness: "{{ slots.name }} Helligkeit auf {{ slots.brightness }} gesetzt"
-      brightness_area: Helligkeit in {{ slots.area }} auf {{ slots.brightness }} gesetzt
-      color: "{{ slots.name }} Farbe auf {{ slots.color }} gesetzt"
-      color_area: Farbe in {{ slots.area }} auf {{ slots.color }} gesetzt
+      brightness: "Helligkeit eingestellt"
+      color: "Farbe auf {{ slots.color }} gesetzt"
```

Comment:
meanwhile I think (_based on the [Response Style Guide](https://developers.home-assistant.io/docs/voice/intent-recognition/style-guide)_) we should not repeat any input data (_area, name, state, color,..._)

Reply:
Done.